moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -85,6 +89,9 @@ const ComponentForm = ({
     subcomponentOptions
   );
 
+  const isEditingExistingComponent = initialFormValues !== null;
+  const isSignalComponent = internalTable === "feature_signals";
```

Comment:
Another payoff from being able to inspect the table that a component should insert features into! 🙌